### PR TITLE
Social Previews | Update docs for the package

### PR DESCRIPTION
**Pull request by manzoorwanijk**
We made some changes to the package and also added a few new components

## Proposed Changes

* This PR updates the docs for the social-previews package

## Testing Instructions

Just ensure that the docs are correct and also ensure that there are no typos or grammatical errors.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/social-previews/README.md [hunk 1]
```diff
@@ -87,6 +89,36 @@ const tweetTemplate = {
...
 
+An example of LinkedIn preview
+
+```jsx
+import { LinkedInPreview } from '@automattic/social-previews';
+
+<LinkedInPreview
+	jobTitle="Job Title (Company Name)"
+	image="https://url.for.the/image.png"
+	name="Linked Account Name"
```
Comment (danielpost) [inline packages/social-previews/README.md]: I don't think it really matters, but was this supposed to be `LinkedIn Account Name`?

File: packages/social-previews/README.md [hunk 2]
```diff
@@ -87,6 +89,36 @@ const tweetTemplate = {
...
+	profileImage="https://static.licdn.com/sc/h/1c5u578iilxfi4m4dvc4q810q"
+	title="Post title goes here"
+	text="The text of the post goes here."
+/>;
+```
+
+Tumblr preview
+
+```jsx
+import { TumblrLinkPreview } from '@automattic/social-previews';
```
Comment (danielpost) [inline packages/social-previews/README.md:110]: I only just realised this, but is there a reason this is called `TumblrLinkPreview` and not `TumblrPreview`? The other ones are called `FacebookPreview`, `TwitterPreview` and `LinkedInPreview` right?

**Review by danielpost — APPROVED** [review]
LGTM! Just added some comments that you're free to ignore.

**Review by danielpost — APPROVED** [review]
LGTM! Just added some comments that you're free to ignore.

File: packages/social-previews/README.md (hunk 1 above)
Comment (manzoorwanijk) [reply packages/social-previews/README.md]: Fixed. Thanks

File: packages/social-previews/README.md (hunk 2 above)
Comment (manzoorwanijk) [reply packages/social-previews/README.md:110]: Yeah, that's another inconsistency I want to fix in another PR.

**Pull request merged**